client/lib/local-storage-polyfill/index.js

```diff
@@ -1,27 +0,0 @@
-import debugFactory from 'debug';
-import localStorageBypass from '../local-storage-bypass';
```

Comment:
I wish we could get rid of this one too. It's like an in-memory polyfill that seems to be used only for setting up support sessions right now. 

I think that's the wrong abstraction; instead of polyfilling local storage, we should rather hook into how the actual local storage data is used (the global Redux store) and do any needed alterations there. 

This makes sense to be left for a follow-up PR of course, since it would require additional testing to ensure support sessions continue to work correctly.

Reply:
> I wish we could get rid of this one too. It's like an in-memory polyfill that seems to be used only for setting up support sessions right now.

Yeah I had spent some time considering whether it could be removed, and tend to agree that we would ideally not be working with `localStorage` directly in areas where we need to have this support session enabled.

I'll make a note to follow-up on better understanding how we're relying on `localStorage` for support sessions.

Reply:
Thanks, follow-up sounds good 👍